### ISSUE TITLE
Intermittent ChannelFull test failure

### ISSUE THREAD

**Issue by carltongibson** [django/channels_redis#229]
e.g. https://github.com/django/channels_redis/runs/1328351504?check_suite_focus=true#step:6:105

This is an artefact: not consistent on run or version. 

Need to make the test more reliable.

**Comment by carltongibson** [comment]
Going to note which test(s) fail(s). (If it's always the same one...)

https://github.com/django/channels_redis/runs/2919887893?check_suite_focus=true
```
FAILED tests/test_core.py::test_receive_cancel
```

**Comment by carltongibson** [comment]
https://github.com/django/channels_redis/pull/262/checks?check_run_id=3016793723

```
FAILED tests/test_core.py::test_receive_cancel - channels.exceptions.ChannelFull
```

**Comment by carltongibson** [comment]
https://github.com/django/channels_redis/pull/262/checks?check_run_id=3017259693

```
FAILED tests/test_core.py::test_receive_cancel - channels.exceptions.ChannelFull
```


Looks like doubling the capacity in `test_receive_cancel` might be required here.

**Comment by carltongibson** [comment]
```
tests/test_core.py::test_receive_cancel - channels.exceptions.ChannelFull
```

https://github.com/django/channels_redis/pull/274/checks?check_run_id=3492323345

Ok, it's looks like it's always the same test. 

Just bumping the limit is probably worth a go.

**Comment by revoteon** [comment]
I went ahead and did some debugging in brute force way. I removed `ChannelFull` exception and instead printed out values of capacity and current count to see how far it can overflow with the tests above. Then I ran the test 100 times. The count never exceeded 15.  

I bumped capacity to of channel layer to 20 in `test_receive_cancel` - both in `test_core.py` and in `test_sentinel.py`. Then I ran tests again 100 times. I didn't get any exception.

So I think your suspicion is correct; just bumping the limit twice to 20 seem to resolve issue for both of these tests.
